pilot/pkg/model/authorization.go

```diff
@@ -191,6 +196,23 @@ func (policy *AuthorizationPolicies) RoleToBindingsForNamespace(ns string) map[s
 	return rolesAndBindings.RoleNameToBindings
 }
 
+// RoleForNameAndNamespace returns a ServiceRole from this namespace, given the ServiceRole name.
+// This function always return a non nil struct instance.
+func (policy *AuthorizationPolicies) RoleForNameAndNamespace(ns, roleName string) *rbacproto.ServiceRole {
```

Comment:
The function name has name first and then namespace, so change the argument list to match this order.

Also change the comment: RoleForNameAndNamespace returns a ServiceRole with given name and namespace.

Reply:
Thanks for spotting it. Changed.